doc/libghdl/Initialization.rst

```diff
@@ -0,0 +1,21 @@
+.. _LIB:Initialization:
+
+Initialization
+##############
+
+To be able to use `libghdl`, it first must be initialized.
+
+* Why does it must be initialized?
```

Comment:
There are many data that must be set up, like the name table.  You cannot do anything useful without it.  And then there are also some higher levels constructs that should also be initialized, like the content of the ieee library.